automobile/scenario-creation-tutorial.md

```diff
@@ -96,6 +44,34 @@ You should be able to open the generated world file directly in Webots:
 ![osm_tutorial_import2.png](images/osm_tutorial_import2.png)
 %end
 
+It is recommended at this stage to perform a manual check of the imported roads and crossroads,
+to erase possible issues done by the importer.
+A good strategy could be fix the OSM data typically to add/remove wayPoints and import again the simulation.
+
+
+### Generate the SUMO network files
+
+We can also use the previously downloaded map to generate the SUMO network file. You need to use the [netconvert](http://sumo.dlr.de/wiki/NETCONVERT) utility for this:
+
+```sh
+cd $WEBOTS_HOME/projects/automobile/resources/SUMO_exporter
+python exporter.py --input=$WBT_PROJECT_PATH/worlds/myMap.wbt --output=$WBT_PROJECT_PATH/worlds/myMap_net
+$WEBOTS_HOME/projects/automobile/resources/bin/netconvert --node-files=$WBT_PROJECT_PATH/worlds/myMap_net/sumo.nod.xml --edge-files=$WBT_PROJECT_PATH/worlds/myMap_net/sumo.edg.xml --output-file=$WBT_PROJECT_PATH/worlds/myMap_net/sumo.net.xml
+```
+
+It is recommended at this stage to perform a manual check of the SUMO network file.
+To do this, open the `sumo.net.xml` file in SUMO `netedit`:
+
+```sh
+$WEBOTS_HOME/projects/automobile/resources/bin/netedit $WBT_PROJECT_PATH/worlds/myMap_net/sumo.net.xml
+```
+
+You can then generate the route file, SUMO provides several ways to generate route files. You may for example generate a [flow file](http://sumo.dlr.de/wiki/Definition_of_Vehicles,_Vehicle_Types,_and_Routes) and then use [duarouter](http://sumo.dlr.de/wiki/DUAROUTER) to generate the route file for you:
+
+```sh
+./duarouter --flows $WBT_PROJECT_PATH/worlds/myMap_net/sumo.flow.xml --net-file $WBT_PROJECT_PATH/worlds/myMap_net/sumo.net.xml --output-file $WBT_PROJECT_PATH/worlds/myMap_net/sumo.rou.xml
+```
```

Comment:
We should also replace `./duarouter` by `$WEBOTS_HOME/projects/automobile/resources/sumo/bin/duarouter`